.circleci/config.yml

```diff
@@ -9,8 +9,8 @@
 
 ## IMPORTANT
 # If you change the `default_docker_image` version, also change the `cache_key` version
-var_1: &default_docker_image circleci/node:12.18.3
-var_2: &browsers_docker_image circleci/node:12.18.3-browsers
+var_1: &default_docker_image circleci/node:12.22.7
```

Comment:
This is a version of node which supports m1?

Reply:
The architect e2e rule relies on host's version of chrome. In order for ci to pass I need a matching chrome version in package.json and whatever this docker container contains.

Reply:
The version of chrome in the previous container version didn't have an m1 release.